tests/VectorException/enc/sigill_handling.c

```diff
@@ -169,18 +187,18 @@ int enc_test_sigill_handling(
         return -1;
     }
 
-    if (!test_unsupported_cpuid_leaf(OE_CPUID_EXTENDED_CPUID_LEAF))
+    if (!test_unsupported_cpuid_leaf(0x14))
```

Comment:
Can you also add a sanity check that `oe_is_emulated_cpuid_leaf` returns false for `0x14`  and log that? It would help with debugging if the assumption that `0x14` is an unsupported leaf is violated in the future.